boards/arm/google_kukui/pinmux.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2017 RnDity Sp. z o.o.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <sys/sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+static const struct pin_config pinconf[] = {
+#ifdef CONFIG_UART_1
+	{STM32_PIN_PA9, STM32F0_PINMUX_FUNC_PA9_USART1_TX},
+	{STM32_PIN_PA10, STM32F0_PINMUX_FUNC_PA10_USART1_RX},
+#endif	/* CONFIG_UART_1 */
+#ifdef CONFIG_I2C_1
```

Comment:
Since board dts is only enabling UART, please remove I2C and SPI pinmux for now.

Reply:
Done